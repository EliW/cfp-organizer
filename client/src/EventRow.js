import React from 'react';

export const EventRow = ({ name, url, location, city, country, countryCode, date, cfpClose, cfpUrl, eventTags }) => (
  <div className="EventRow">
    <div className="EventRow-cell"><a href={url}>{name}</a></div>
    <div className="EventRow-cell EventRow-cell--location">{location} <span className='u-small'>{city}, {country}, {countryCode}</span></div>
    <div className="EventRow-cell">{date}</div>
    <div className="EventRow-cell"><a href={cfpUrl}>{cfpClose}</a></div>
    <div className="EventRow-cell">{eventTags}</div>
  </div>
);