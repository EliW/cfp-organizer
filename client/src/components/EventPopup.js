import React from 'react';

export const EventPopup = ({ event }) => (
  <React.Fragment>
    <a href={event.url} target="_blank">{event.name}</a> - { event.date } <br/>
    { event.city }, { event.country } <br/>
    <a href={event.cfpUrl} target="_blank">CFP Closes on {event.cfpClose}</a>
  </React.Fragment>
);