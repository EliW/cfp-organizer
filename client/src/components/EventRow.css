.SvgButton {
  height: 30px;
  width: 30px;
  border: none;
  background: none;
  cursor: pointer;
  margin-right: 10px;
}

.SvgButton:focus {
  outline: 3px auto #0074d9;
}

.SvgButton::-moz-focus-inner {
  border: 0;
}

.SvgButton svg {
  outline: none;
}

.u-srOnly {
  height: 1px;
  overflow: hidden;
  position: absolute;
  width: 1px;
}

.EventRow-button {
  border: none;
  display: block;
  font-size: 1em;
  width: 100%;
}
