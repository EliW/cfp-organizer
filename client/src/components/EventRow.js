import React from 'react';

export const EventRow = ({ className, rowClassName, event }) => (
  <div className={`${className} EventRow`}>
    <div className={rowClassName}><a href={event.url}>{event.name}</a></div>
    <div className={`${rowClassName} Table-cell--location`}>
      {event.location} 
      <span className='u-small'>{event.city}, {event.country}, {event.countryCode}, {event.continent}</span>
    </div>
    <div className={rowClassName}>{event.date}</div>
    <div className={rowClassName}><a href={event.cfpUrl}>{event.cfpClose}</a></div>
    <div className={`${rowClassName} u-small`}>{event.eventTags}</div>
  </div>
);