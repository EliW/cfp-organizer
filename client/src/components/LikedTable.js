import React from "react";
import { Table } from "./Table.js";

export const LikedTable = props => {
  const likedEventsFromStorage = localStorage.getItem(
    "cfpOrganizer-likedEvents"
  );
  let likedEvents = {};
  if (likedEventsFromStorage) {
    likedEvents = {
      ...JSON.parse(localStorage.getItem("cfpOrganizer-likedEvents"))
    };
  }

  const data = props.data.filter(event => likedEvents[event.name]);

  return <Table {...props} data={data} />;
};
