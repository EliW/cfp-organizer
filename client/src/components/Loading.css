.Loading {
  color: #666;
  margin: 6em;
  text-align: center;
}

.Loading-spinner {
  display: inline-block;
  position: relative;
  width: 64px;
  height: 64px;
}
.Loading-spinner div {
  position: absolute;
  width: 5px;
  height: 5px;
  background: #aaa;
  border-radius: 50%;
  animation: Loading 1.2s linear infinite;
}
.Loading-spinner .one {
  animation-delay: 0s;
  top: 29px;
  left: 53px;
}
.Loading-spinner .two {
  animation-delay: -0.1s;
  top: 18px;
  left: 50px;
}
.Loading-spinner .three {
  animation-delay: -0.2s;
  top: 9px;
  left: 41px;
}
.Loading-spinner .four {
  animation-delay: -0.3s;
  top: 6px;
  left: 29px;
}
.Loading-spinner .five {
  animation-delay: -0.4s;
  top: 9px;
  left: 18px;
}
.Loading-spinner .six {
  animation-delay: -0.5s;
  top: 18px;
  left: 9px;
}
.Loading-spinner .seven {
  animation-delay: -0.6s;
  top: 29px;
  left: 6px;
}
.Loading-spinner .eight {
  animation-delay: -0.7s;
  top: 41px;
  left: 9px;
}
.Loading-spinner .nine {
  animation-delay: -0.8s;
  top: 50px;
  left: 18px;
}
.Loading-spinner .ten {
  animation-delay: -0.9s;
  top: 53px;
  left: 29px;
}
.Loading-spinner .eleven {
  animation-delay: -1s;
  top: 50px;
  left: 41px;
}
.Loading-spinner .twelve {
  animation-delay: -1.1s;
  top: 41px;
  left: 50px;
}
@keyframes Loading {
  0%, 20%, 80%, 100% {
    transform: scale(1);
  }
  50% {
    transform: scale(1.5);
  }
}
