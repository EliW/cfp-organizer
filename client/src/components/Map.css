.Map {
  padding-bottom: 20px;
  height: calc(100% - 200px); 
  margin: 0 20px;
}