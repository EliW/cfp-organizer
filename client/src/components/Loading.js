import React from 'react';
import './Loading.css';

export const Loading = () => (
  <React.Fragment>
    <div className="Loading">
      <div className="Loading-spinner">
        <div className="one"></div>
        <div className="two"></div>
        <div className="three"></div>
        <div className="four"></div>
        <div className="five"></div>
        <div className="six"></div>
        <div className="seven"></div>
        <div className="eight"></div>
        <div className="nine"></div>
        <div className="ten"></div>
        <div className="eleven"></div>
        <div className="twelve"></div>
      </div>
      <p>Loading...</p>
    </div>
  </React.Fragment>
);