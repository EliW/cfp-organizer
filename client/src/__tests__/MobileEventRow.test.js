/* eslint-env jest */
import * as React from 'react'
import { shallow } from 'enzyme'
import { EventRow } from '../components/EventRow'

let subject
let event
let isActive
let onClick

function loadSubject(props = {}) {
  event = {
    name: 'name',
    url: 'http://url.com',
    date: 'September 30, 2018',
    cfpClose: 'September 30, 2018',
    cfpUrl: 'http://url.com',
    location: 'Chicago, IL',
    eventTags: 'Javascript, Ruby, Tech',
    city: 'Chicago',
    country: 'United States',
    countryCode: 'US',
    contintent: 'North America',
    lat: 41,
    lng: -55,
  }

  onClick = jest.fn()
  isActive = false

  subject = shallow(<EventRow
    className='EventRow'
    cellClassName='EventRow-cell'
    event={event}
    isActive={isActive}
    onClick={onClick}
    {...props}
  />)
}

describe('when not active', () => {
  it('renders properly', () => {
    loadSubject()
    expect(subject).toMatchSnapshot()
  })
})

describe('when active', () => {
  it('renders properly', () => {
    loadSubject({isActive: true})
    expect(subject).toMatchSnapshot()
  })
})