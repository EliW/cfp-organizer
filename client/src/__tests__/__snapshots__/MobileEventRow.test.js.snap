// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when active renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    cellClassName="EventRow-cell"
    className="EventRow"
    event={
        Object {
            "cfpClose": "September 30, 2018",
            "cfpUrl": "http://url.com",
            "city": "Chicago",
            "contintent": "North America",
            "country": "United States",
            "countryCode": "US",
            "date": "September 30, 2018",
            "eventTags": "Javascript, Ruby, Tech",
            "lat": 41,
            "lng": -55,
            "location": "Chicago, IL",
            "name": "name",
            "url": "http://url.com",
          }
    }
    isActive={true}
    onClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="EventRow-cell"
>
          <a
                    href="http://url.com"
                    target="_blank"
          >
                    name
          </a>
</div>,
        <div
          className="EventRow-cell Table-cell--location"
>
          Chicago, IL
          <span
                    className="u-small"
          >
                    Chicago
                    , 
                    , 
                    United States
                    , 
          </span>
</div>,
        <div
          className="EventRow-cell"
>
          September 30, 2018
</div>,
        <div
          className="EventRow-cell"
>
          <a
                    href="http://url.com"
                    target="_blank"
          >
                    September 30, 2018
          </a>
</div>,
        <div
          className="EventRow-cell u-small"
>
          Javascript, Ruby, Tech
</div>,
      ],
      "className": "EventRow EventRow",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="http://url.com"
            target="_blank"
>
            name
</a>,
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "name",
            "href": "http://url.com",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "name",
          "type": "a",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Chicago, IL",
            <span
              className="u-small"
>
              Chicago
              , 
              , 
              United States
              , 
</span>,
          ],
          "className": "EventRow-cell Table-cell--location",
        },
        "ref": null,
        "rendered": Array [
          "Chicago, IL",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Chicago",
                ", ",
                undefined,
                ", ",
                "United States",
                ", ",
                undefined,
              ],
              "className": "u-small",
            },
            "ref": null,
            "rendered": Array [
              "Chicago",
              ", ",
              undefined,
              ", ",
              "United States",
              ", ",
              undefined,
            ],
            "type": "span",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "September 30, 2018",
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": "September 30, 2018",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="http://url.com"
            target="_blank"
>
            September 30, 2018
</a>,
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "September 30, 2018",
            "href": "http://url.com",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "September 30, 2018",
          "type": "a",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Javascript, Ruby, Tech",
          "className": "EventRow-cell u-small",
        },
        "ref": null,
        "rendered": "Javascript, Ruby, Tech",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="EventRow-cell"
>
            <a
                        href="http://url.com"
                        target="_blank"
            >
                        name
            </a>
</div>,
          <div
            className="EventRow-cell Table-cell--location"
>
            Chicago, IL
            <span
                        className="u-small"
            >
                        Chicago
                        , 
                        , 
                        United States
                        , 
            </span>
</div>,
          <div
            className="EventRow-cell"
>
            September 30, 2018
</div>,
          <div
            className="EventRow-cell"
>
            <a
                        href="http://url.com"
                        target="_blank"
            >
                        September 30, 2018
            </a>
</div>,
          <div
            className="EventRow-cell u-small"
>
            Javascript, Ruby, Tech
</div>,
        ],
        "className": "EventRow EventRow",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="http://url.com"
              target="_blank"
>
              name
</a>,
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "name",
              "href": "http://url.com",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "name",
            "type": "a",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Chicago, IL",
              <span
                className="u-small"
>
                Chicago
                , 
                , 
                United States
                , 
</span>,
            ],
            "className": "EventRow-cell Table-cell--location",
          },
          "ref": null,
          "rendered": Array [
            "Chicago, IL",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Chicago",
                  ", ",
                  undefined,
                  ", ",
                  "United States",
                  ", ",
                  undefined,
                ],
                "className": "u-small",
              },
              "ref": null,
              "rendered": Array [
                "Chicago",
                ", ",
                undefined,
                ", ",
                "United States",
                ", ",
                undefined,
              ],
              "type": "span",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "September 30, 2018",
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": "September 30, 2018",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="http://url.com"
              target="_blank"
>
              September 30, 2018
</a>,
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "September 30, 2018",
              "href": "http://url.com",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "September 30, 2018",
            "type": "a",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Javascript, Ruby, Tech",
            "className": "EventRow-cell u-small",
          },
          "ref": null,
          "rendered": "Javascript, Ruby, Tech",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;

exports[`when not active renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    cellClassName="EventRow-cell"
    className="EventRow"
    event={
        Object {
            "cfpClose": "September 30, 2018",
            "cfpUrl": "http://url.com",
            "city": "Chicago",
            "contintent": "North America",
            "country": "United States",
            "countryCode": "US",
            "date": "September 30, 2018",
            "eventTags": "Javascript, Ruby, Tech",
            "lat": 41,
            "lng": -55,
            "location": "Chicago, IL",
            "name": "name",
            "url": "http://url.com",
          }
    }
    isActive={false}
    onClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="EventRow-cell"
>
          <a
                    href="http://url.com"
                    target="_blank"
          >
                    name
          </a>
</div>,
        <div
          className="EventRow-cell Table-cell--location"
>
          Chicago, IL
          <span
                    className="u-small"
          >
                    Chicago
                    , 
                    , 
                    United States
                    , 
          </span>
</div>,
        <div
          className="EventRow-cell"
>
          September 30, 2018
</div>,
        <div
          className="EventRow-cell"
>
          <a
                    href="http://url.com"
                    target="_blank"
          >
                    September 30, 2018
          </a>
</div>,
        <div
          className="EventRow-cell u-small"
>
          Javascript, Ruby, Tech
</div>,
      ],
      "className": "EventRow EventRow",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="http://url.com"
            target="_blank"
>
            name
</a>,
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "name",
            "href": "http://url.com",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "name",
          "type": "a",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Chicago, IL",
            <span
              className="u-small"
>
              Chicago
              , 
              , 
              United States
              , 
</span>,
          ],
          "className": "EventRow-cell Table-cell--location",
        },
        "ref": null,
        "rendered": Array [
          "Chicago, IL",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Chicago",
                ", ",
                undefined,
                ", ",
                "United States",
                ", ",
                undefined,
              ],
              "className": "u-small",
            },
            "ref": null,
            "rendered": Array [
              "Chicago",
              ", ",
              undefined,
              ", ",
              "United States",
              ", ",
              undefined,
            ],
            "type": "span",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "September 30, 2018",
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": "September 30, 2018",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="http://url.com"
            target="_blank"
>
            September 30, 2018
</a>,
          "className": "EventRow-cell",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "September 30, 2018",
            "href": "http://url.com",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "September 30, 2018",
          "type": "a",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Javascript, Ruby, Tech",
          "className": "EventRow-cell u-small",
        },
        "ref": null,
        "rendered": "Javascript, Ruby, Tech",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="EventRow-cell"
>
            <a
                        href="http://url.com"
                        target="_blank"
            >
                        name
            </a>
</div>,
          <div
            className="EventRow-cell Table-cell--location"
>
            Chicago, IL
            <span
                        className="u-small"
            >
                        Chicago
                        , 
                        , 
                        United States
                        , 
            </span>
</div>,
          <div
            className="EventRow-cell"
>
            September 30, 2018
</div>,
          <div
            className="EventRow-cell"
>
            <a
                        href="http://url.com"
                        target="_blank"
            >
                        September 30, 2018
            </a>
</div>,
          <div
            className="EventRow-cell u-small"
>
            Javascript, Ruby, Tech
</div>,
        ],
        "className": "EventRow EventRow",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="http://url.com"
              target="_blank"
>
              name
</a>,
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "name",
              "href": "http://url.com",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "name",
            "type": "a",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Chicago, IL",
              <span
                className="u-small"
>
                Chicago
                , 
                , 
                United States
                , 
</span>,
            ],
            "className": "EventRow-cell Table-cell--location",
          },
          "ref": null,
          "rendered": Array [
            "Chicago, IL",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Chicago",
                  ", ",
                  undefined,
                  ", ",
                  "United States",
                  ", ",
                  undefined,
                ],
                "className": "u-small",
              },
              "ref": null,
              "rendered": Array [
                "Chicago",
                ", ",
                undefined,
                ", ",
                "United States",
                ", ",
                undefined,
              ],
              "type": "span",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "September 30, 2018",
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": "September 30, 2018",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="http://url.com"
              target="_blank"
>
              September 30, 2018
</a>,
            "className": "EventRow-cell",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "September 30, 2018",
              "href": "http://url.com",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "September 30, 2018",
            "type": "a",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Javascript, Ruby, Tech",
            "className": "EventRow-cell u-small",
          },
          "ref": null,
          "rendered": "Javascript, Ruby, Tech",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
